src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java

```diff
@@ -11,7 +11,6 @@
  */
 package org.fundacion.jala.converter.models.parameter;
 
-import org.fundacion.jala.converter.service.AudioConverter;
 import org.springframework.stereotype.Service;
 
 @Service
```

Comment:
This would require some changes on the arquitecture of the program, because there are several parameters working like these with the converters. On a new PR a person could work on fixing this problem.